hub/values.yaml

```diff
@@ -141,8 +148,68 @@ jupyterhub:
               line = f.readline()
           return members
 
+        def course_error_messages(courses):
+          return map(lambda x: x.get('message', ''), courses['errors'])
+
+        def course_filter(courses, enrollment_type, attr='id'):
+          '''Filter course attributes by enrollment type.'''
+          attrs = []
+          for course in courses:
+            if 'enrollments' not in course: continue
+            for enrollment in course['enrollments']:
+              if enrollment['type'] == enrollment_type:
+                attrs.append(course[attr]); break
```

Comment:
How do you feel about not adding multiple statements on the same line? I generally find that most python projects don't do that, and it always makes me feel like i'm reading C :)

This applies for both semicolons in stuff like this, and stuff like `             if 'enrollments' not in course: continue`

Reply:
Sure, can do. Apparently I like fitting code into 80x24. :)